ChangeLog.d/fix-pkgconfig-static-linking-windows.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix:
```

Comment:
(Note: if this gets merged before the 3.6.0 code freeze, we need to remove the changelog entry, since pkgconfig is new in 3.6. But at this point it's very unlikely to make it into 3.6.0.)